homestar-runtime/src/logger.rs

```diff
@@ -7,6 +7,7 @@ use tracing_subscriber::{layer::SubscriberExt as _, prelude::*, EnvFilter};
 
 const LOG_FILE: &str = "homestar.log";
 const DIRECTIVE_EXPECT: &str = "Invalid tracing directive";
+const EVERY_CLI: &str = "EVERY_CLI";
```

Comment:
minor: I'd probably write a small comment at the top on any env vars and why they're generally here.

Reply:
Yep, adding a comment 👍